.gitignore

```diff
@@ -104,6 +104,7 @@ src/goto-instrument/goto-instrument.exe
 src/jbmc/jbmc
 src/musketeer/musketeer
 src/musketeer/musketeer.exe
+src/solvers/smt2/smt2_solver
```

Comment:
Even though a later commit disables this: the .exe variant should be added now.

Reply:
done